test/bats/test.bats

```diff
@@ -8,11 +8,25 @@ WAIT_TIME_DEPLOYMENTS=600
 SLEEP_TIME_DEPLOYMENTS=10
 ARC_CLUSTER=${ARC_CLUSTER:-true}
 
+@test "chart version on cluster matches extension tag" {
+    if [ $ARC_CLUSTER == false ]; then
+        skip "arc cluster-specific test"
+    fi
+
+    run wait_for_condition $WAIT_TIME_DEPLOYMENTS $SLEEP_TIME_DEPLOYMENTS "\"$(helm ls -o json --namespace arc-osm-system | jq -r '.[].chart')\" == \"osm-arc-$EXTENSION_TAG\""
+    assert_success
+}
+
 @test "arc-osm-system deployments have succeeded" {
     run wait_for_process $WAIT_TIME_DEPLOYMENTS $SLEEP_TIME_DEPLOYMENTS "kubectl wait --for=condition=available deployment --all --namespace arc-osm-system"
     assert_success
 }
 
+@test "only one osm-controller pod" {
+    run wait_for_condition $WAIT_TIME $SLEEP_TIME "$(kubectl get pods -n arc-osm-system -l app=osm-controller | wc -l) == 2"
```

Comment:
I think this is the same as what I already have. The osm-controller label is specific to the osm-controller pod.

Reply:
This is not the same because you are just checking for any pods in the osm-controller namespace. You should check specifically for the osm-controller pod, again because the new release introduces an osm-injector pod in the same namespace